### ISSUE TITLE
no current rain value

### ISSUE THREAD

**Issue by Diginix** [Bart1909/ioBroker.netatmo-crawler#39]
If I open the map I see 
https://weathermap.netatmo.com//?stationid=70%3Aee%3A50%3A65%3A37%3Ae8
![image](https://user-images.githubusercontent.com/39678667/154664544-74b4439e-1f65-4787-8f38-732acb57eeca.png)
But netatmo-crawler.0.stationData.1.rain value is 0
netatmo-crawler.0.stationData.1.rain_today is 0.9
Same for a second station:
https://weathermap.netatmo.com//?stationid=70%3Aee%3A50%3A03%3A64%3A2c
![image](https://user-images.githubusercontent.com/39678667/154664649-687bc84a-aec9-48fd-b7ef-d8393a1e0d0a.png)
netatmo-crawler.0.stationData.2.rain is 0
netatmo-crawler.0.stationData.2.rain_today is 0.7

adapter: 0.7.1
js-controller: 4.0.12
nodejs: 14.19.0

**Comment by Bart1909** [comment]
I don't know, what causes this issue. The API from Netatmo is returning also was 0 for current rain.
I will have to further analyze, how the webpage gets the current rain value

**Comment by Bart1909** [comment]
It seems, the rain value from the api is now a real time value and not the sum of the last hour.
Because of this, i've added a new state 'rain_lastHour' which gets the sum of the rain in the last hour.
Please try latest version 0.8.x

**Comment by Diginix** [comment]
Thanks! And the realtime value from website will be in netatmo-crawler.0.stationData.0.rain?
Because this was 0 even when website shows a value.
With 0.8 there will be 3 values?
- realtime
- hour
- day

**Comment by Bart1909** [comment]
I've changed nothing on the .rain state. This _should_ be the real time value, I think.

With 0.8 you will see .rain (Real Time), .rain_lastHour, .rain_today and .rain_yesterday

**Comment by Diginix** [comment]
Ah ok, I guess the realtime value isn't shown on the website and comes only with the API.
0.8.1 works, but at the moment there is no rain to check the new state.

**Comment by Bart1909** [comment]
Yep, seems so. I guess, the website collects data from all stations in the visible area and summarizes the data. The network activity suggests that

**Comment by mcm1957** [comment]
no more feedback
